crates/subspace-malicious-operator/src/malicious_bundle_producer.rs

```diff
@@ -269,12 +269,21 @@ where
             }
         };
 
-        let maybe_operator_id = self
-            .consensus_client
-            .runtime_api()
-            .operator_id_by_signing_key(consensus_best_hash, signing_key.clone())?;
+        let mut maybe_operator_id = None;
```

Comment:
if its helpful, I would recommend adding this test runtime api.
We have test specific runtime apis defined that are not implemented in actual runtime. 
Just a thought

Reply:
I’m not sure what you mean here. Do you mean restore the storage in the test runtime, or something else?